.changeset/eight-owls-shop.md

```diff
@@ -0,0 +1,13 @@
+---
+"@comet/admin-icons": minor
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add UI for Impersonation Feature
+
+-   Add indicator to display when impersonation mode is active in `UserHeaderItem`
+-   Add button to allow users to switch on impersonation in the `UserGrid`
+-   Integrate `CrudMoreActionsMenu` in `UserPageToolbar` with an impersonation entry for easy access to this feature.
+-   Add util functions `startPersonation` and `stopPersonation`
```

Comment:
```suggestion
```

These aren't added to the public API, so they shouldn't be in the changeset

Reply:
231ea93cb4ce256f986fbccd7830133d40358774